coralproject/talk
# Allow optionally toggling showing of comment id's with a hotkey

## What does this PR do?

Allows showing of comment ID's on the comment by using the hotkey `Alt + H`.

I use this feature all the time to debug stuff and have been leaving it in a stash that I apply, but it feels like it should maybe just be a feature so devs, moderators, SRE, and admins can simply toggle it on when they need it for debugging the stream or for finding comment ID's quickly from the stream.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes added.

## How do I test this PR?

- Start up Coral
- Go to the stream
- Press `Alt + H` on your keyboard when selecting a comment
- All comments should now show a comment ID above their comment body
- Press `Alt + H` again
- Comments should no longer be showing their comment ID
 
## How do we deploy this PR?

No special considerations.
